app/javascript/components/Place.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const Place = ({place}) => {
+    return (
+        <div>
+            <h5>{place.number}</h5>
+        </div>
+    );
+};
```

Comment:
```suggestion
const Place = ({place}) => (
    <div>
        <h5>{place.number}</h5>
    </div>
);
```

Also please use PropTypes from 'prop-types'; It helps you to avoid some issues in future